esxi/guest-create.go

```diff
@@ -17,7 +17,7 @@ import (
 
 func guestCREATE(c *Config, guest_name string, disk_store string,
 	src_path string, resource_pool_name string, strmemsize string, strnumvcpus string, strvirthwver string, guestos string,
-	boot_disk_type string, boot_disk_size string, virtual_networks [10][3]string,
+	boot_disk_type string, boot_disk_size string, virtual_networks [10][3]string, is_uefi bool,
```

Comment:
Please rename the variable to match the resource name. (boot_firmware)

Reply:
fixed